src/client/src/ui/common/popout/reactPopoutService.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {PopoutOptions} from './';
+import {logger} from '../../../system';
+
+import './popoutRegion.scss';
+
+let POPOUT_CONTAINER_ID = 'popout-content-container';
+
+let _log:logger.Logger = logger.create('ReactPopoutService');
+
+export default class ReactPopoutService {
+  openPopout(options:PopoutOptions, view:React.Component) {
+    let popoutContainer;
+    let windowOptionsString = this._getWindowOptionsString(options.windowOptions);
+    _log.debug(`Opening child window url:${options.url},title:${options.title}`);
+    let childWindow = window.open(options.url, options.title, windowOptionsString);
+    let onloadHandler = () => {
+      _log.debug(`Popout window loading`);
+      childWindow.document.title = options.title;
+      popoutContainer = childWindow.document.createElement('div');
+      popoutContainer.id = POPOUT_CONTAINER_ID;
+      childWindow.document.body.appendChild(popoutContainer);
```

Comment:
our body route has `div#root` in it, should we not append to that like we do in the main app? else, DOM ends up as `<div id=root></div><div id=${POPOUT_CONTAINER_ID}>...</div>`